dndwiki/classes/class-gunslinger.xml

```diff
@@ -0,0 +1,886 @@
+    </description>
+    <sheet>
+      <description>You may use your reaction to make a ranged weapon attack with a firearm you are holding against the marked creature when it attacks. This attack happens before the marked creature hits.</description>
+    </sheet>
+  </element>
+  <element name="Master Marksman" type="Archetype Feature" source="D&amp;D Wiki" id="ID_WIKI_ARCHETYPE_FEATURE_MASTER_MARKSMAN">
+   <description>
+      <p>Starting at 20th level, your Sniper's Mark feature becomes second nature to you. You now regain all uses of your Sniper's Mark feature when you finish a short or long rest.</p>
+    </description>
+    <sheet>
+      <description>You regain all uses of your Sniper's Mark feature when you finish a short rest.</description>
+    </sheet>
+  </element>
+
+  <!-- CQC Archetype -->
+  <element name="Close Quarter Combatant" type="Archetype" source="D&amp;D Wiki" id="ID_WIKI_ARCHETYPE_CQC">
+    <supports>Gunslinger Archetype</supports>
+    <description>
+      <p>Close Quarter Combatant experts prefer to be at the front line of combat, dealing damage and taking hits in place of other teammates.</p>
+      <div element="ID_WIKI_ARCHETYPE_FEATURE_ARMORED_BRUTE"/>
+      <div element="ID_WIKI_ARCHETYPE_FEATURE_CQC_TRAINING"/>
+      <div element="ID_WIKI_ARCHETYPE_FEATURE_COMBAT_CONDITIONING"/>
+      <div element="ID_WIKI_ARCHETYPE_FEATURE_CROWD_CONTROL"/>
+      <div element="ID_WIKI_ARCHETYPE_FEATURE_RIOT_SHIELD"/>
+      <div element="ID_WIKI_ARCHETYPE_FEATURE_MASTER_COMBATANT" />
+    </description>
+    <sheet>
+      <description>Close Quarter Combatant experts prefer to be at the front line of combat.</description>
+    </sheet>
+    <rules>
+      <grant type="Archetype Feature" name="ID_WIKI_ARCHETYPE_FEATURE_ARMORED_BRUTE" level="3"/>
+      <grant type="Archetype Feature" name="ID_WIKI_ARCHETYPE_FEATURE_CQC_TRAINING" level="3"/>
+      <grant type="Archetype Feature" name="ID_WIKI_ARCHETYPE_FEATURE_COMBAT_CONDITIONING" level="7"/>
+      <grant type="Archetype Feature" name="ID_WIKI_ARCHETYPE_FEATURE_CROWD_CONTROL" level="13"/>
+      <grant type="Archetype Feature" name="ID_WIKI_ARCHETYPE_FEATURE_RIOT_SHEILD" level="18"/>
+      <grant type="Archetype Feature" name="ID_WIKI_ARCHETYPE_FEATURE_MASTER_COMBATANT" level="20"/>
+    </rules>
+  </element>
+  <element name="Armored Brute" type="Archetype Feature" source="D&amp;D Wiki" id="ID_WIKI_ARCHETYPE_FEATURE_ARMORED_BRUTE">
+    <description>
+      <p>Beginning at 3rd level, you gain +1 to your AC as long as you are wearing armor. The bonus to AC while you are wearing armor increases by 1 at 15th level (+2).</p>
+    </description>
+    <sheet display="false">
+      <description /> 
+    </sheet>
+    <rules>
+      <stat name="ac:misc" value="1" bonus="armored brute" equipped="[armor:any]" alt="Armored Brute" />
+      <stat name="ac:misc" value="1" bonus="armored brute" equipped="[armor:any]" alt="Armored Brute" level="15" />
```

Comment:
```suggestion
      <stat name="ac:misc" value="2" bonus="armored brute" equipped="[armor:any]" alt="Armored Brute" level="15" />
```
Since it's the same bonus, they will replace each other